src/pages/search.tsx

```diff
@@ -158,17 +158,36 @@ const Search: NextPage<SearchProps> = ({ translations, chaptersData }): JSX.Elem
     [],
   );
 
+  // a ref to know weather this is the initial search request made when the user loads the page or not
+  const isInitialSearch = useRef(true);
+
   // listen to any changes in the API params and call BE on change.
   useEffect(() => {
     // only when the search query has a value we call the API.
     if (debouncedSearchQuery) {
-      getResults(debouncedSearchQuery, currentPage, selectedTranslations, selectedLanguages);
+      // we don't want to reset pagination when the user reloads the page with a ?page={number} in the url query
+      if (!isInitialSearch.current) setCurrentPage(1);
+
+      getResults(
+        debouncedSearchQuery,
+        // if it is the initial search request, use the page number in the url, otherwise, reset it
+        isInitialSearch.current ? currentPage : 1,
+        selectedTranslations,
+        selectedLanguages,
+      );
+
+      // if it was the initial request, update the ref
+      if (isInitialSearch.current) isInitialSearch.current = false;
```

Comment:
ditto. We should use `{}`.

Reply:
Maybe we should add an eslint rule ?

Reply:
Yes would be great to do so in another PR! Thanks a lot